Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml

```diff
@@ -0,0 +1,346 @@
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - ClothingBackpackHolding
+        - ClothingBackpackMessengerHolding
+        - ClothingBackpackDuffelHolding
+        - ClothingShoesBootsMagAdv
+        - AdvMopItem
+        - RGBStaff
+        - WeaponAdvancedLaser
+        - AmmoTechFabCircuitboard
+        - PlushieArachind
+        - ClothingMaskArcadia
+        - ClothingBeltChiefEngineerFilled
+        - CowToolboxFilled
+        - ClothingShoesBootsMagCombatFilled
+        - ClothingShoesBootsMagGaloshes
+        - Hypospray
+        - Katana
+        - WeaponLaserCannon
+        - MuteToxinChemistryBottle
+        - ClothingOuterHardsuitCap
+        - ClothingShoesBootsMagBlinding
+        - TurboItemRechargerCircuitboard
+        - WeaponRevolverDeckard
+        - MedicatedSuture
+        - FoodPizzaArnold
+        - PinpointerUniversal
+        - Claymore
+      rareChance: 0.3
+
+- type: entity
+  id: RandomRarelegalLoot
+  name: Random Rare Legal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetUSA
+    - type: RandomSpawner
+      prototypes:
+        - ClothingBeltSheathFilled
+        - SpaceCash10000
+        - WeaponAntiqueLaser
+        - WeaponForceGun
+        - PenCentcom
+        - ClothingOuterArmorPodWars
+        - ClothingHeadHelmetPodWars
+        - ClothingOuterCoatSpaceAsshole
+        - UnholyHalberd
+        - ClockworkShield
+        - MirrorShield
+        - BedsheetUSA
+        - HypoMini
+        - HypoMiniLimitedEdition
+        - ClothingBackpackHolding
+        - ClothingShoesBootsMagAdv
+        - OreBagOfHolding
+        - ClothingShoesBootsSpeed
+        - ClothingBackpackDuffelHolding
+        - ClothingBackpackMessengerHolding
+        - MedkitCombatFilled
+        - TrashBagOfHolding
+        - AdvMopItem
+        - HandheldCrewMonitor
+        - WeaponAdvancedLaser
+        - VariedXenoArtifactItem
+        - ClothingMaskArcadia
+        - ClothingShoesBootsMagCombatFilled
+        - WeaponLaserCannon
+        - ClothingEyesHudMedSec
+        - PowerCellMicroreactor
+        - MuteToxinChemistryBottle
+        - SmokeGrenade
+        - ClothingShoesBootsMagBlinding
+        - TechnologyDiskRare
+        - Truncheon
+        - TurboItemRechargerCircuitboard
+        - ResearchDisk10000
+        - MedicatedSuture
+        - RegenerativeMesh
+        - FoodPizzaArnold
+        - PinpointerUniversal
+      chance: 1
+
+- type: entity
+  id: RandomEventIllegalLoot 
+  name: Random Illegal Loot
+  suffix: Spawner 
+  parent: MarkerBase
+  components:
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/bedsheets.rsi
+          state: sheetsyndie
+    - type: RandomSpawner
+      prototypes:
+      - DnaScramblerImplanter
+      - CyberPen
+      - CigPackSyndicate
+      - WeaponRevolverPythonAP
```

Comment:
Rare at minimum...

Reply:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/124747282/a9b51802-6fab-4aad-97ee-e32188aca58c)
4 shots to kill a full health death squad in hardsuit.

Reply:
well thats because it ignores Body armor completely so it does True damage to the person it self

Reply:
move to Rare